src/containers/Webmap/Webmap.scss

```diff
@@ -0,0 +1,6 @@
+.leaflet-container {
+  height: 100vh;
```

Comment:
in mobile view, since it takes up the full height of the device, it is quite difficult to scroll down to the bottom

Reply:
Ah, didn't think about that. We could change it to

```
.leaflet-container {
  height: 50vh;
  margin-top: 10px;
  width: 100%;
  z-index: 1;
}

```
and have it look more lie this ?

![like_this](https://user-images.githubusercontent.com/9319623/102696141-1e3ee580-41fa-11eb-8e12-6339274db2d6.PNG)

Also, looking for feedback on the basemap and icons. I just googled bank and check. We can probably do a little better if anyone maybe wants to try creating some custom svgs?

Reply:
Can @thenewboston-developers/designers provide feedback on this? Thank you!

Reply:
I updated the css to make the map smaller, like the screenshot above. I think it looks better that way.